business_requirement_deliverable_project/wizard/generate_projects.py

```diff
@@ -151,7 +152,8 @@ def generate_deliverable_projects(
                 line_project_val = self._prepare_project_vals(
                     line, parent_project)
                 line_project_val.update({
-                    'business_requirement_deliverable_id': line.id
+                    'business_requirement_deliverable_id': line.id,
+                    'business_requirement_id': line.business_requirement_id.id
                 })
                 line_project = project_obj.create(line_project_val)
                 line.linked_project = line_project.id
```

Comment:
@YogeshMahera-SerpentCS any reason to keep this field `linked_project`?

Reply:
These two fields:

https://github.com/OCA/business-requirement/pull/245/files#diff-e9c58cb738de57ebabe8fec326544d7fR16
https://github.com/OCA/business-requirement/pull/245/files#diff-e9c58cb738de57ebabe8fec326544d7fR126

Reply:
yes no need to keep,I had checked it's not usable so removed it.
@victormartinelicocorp

Reply:
@YogeshMahera-SerpentCS please note that you haven't.

https://github.com/OCA/business-requirement/pull/245/files#diff-e9c58cb738de57ebabe8fec326544d7fR16
https://github.com/OCA/business-requirement/pull/245/files#diff-e9c58cb738de57ebabe8fec326544d7fR126